src/main/java/de/uhd/ifi/se/decision/management/jira/persistence/ActiveObjectStrategy.java

```diff
@@ -75,7 +80,8 @@ public Boolean doInTransaction() {
 
 	@Override
 	public boolean deleteLink(Link link, ApplicationUser user) {
-		return GenericLinkManager.deleteGenericLink("a"+link.getSourceElement().getId(), "a"+link.getDestinationElement().getId());
+		return GenericLinkManager.deleteGenericLink("a" + link.getSourceElement().getId(),
```

Comment:
As above.. The string literal "a" and similar should be realy constants with nice/telling names. I will not repeat this comment anymore.

Reply:
This will be part of future work to get independent of the id prefixes that indicate documentation location.